src/phx/strategy/random/main.py

```diff
@@ -11,7 +11,7 @@
 from phx.fix.model.auth import FixAuthenticationMethod
 
 # internal Phoenix FIX example library
-from phx.strategy.random import RandomStrategy
+from src.phx.strategy.random import RandomStrategy
```

Comment:
This is wrong and MUST stay without the source folder reference "src".
You have to read the README.md and do it that way as PyCharm has to know the source folder. That is the only correct way to do it. 

Using the project without proper source folder configuration is wrong and conflicts with the "package name" structure of Python: If the package gets installed it will not have the source folder anymore!